setup/README.md

```diff
@@ -56,81 +56,76 @@ Before you can finally set up Connaisseur, you will need to configure its connec
+      -----BEGIN PUBLIC KEY-----
+      ...
+      -----END PUBLIC KEY-----
+  - name: app1
+    key: | 
+      -----BEGIN PUBLIC KEY-----
+      ...
+      -----END PUBLIC KEY-----
+  selfsignedCert: | 
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
   auth:
-    enabled: true
-    # enter user/pass directly
-    # these are placeholders and should be changed!
     user: notaryuser
-    password: Password123
-    # or use a predefined secret, which needs the fields 'NOTARY_USER'
-    # and 'NOTARY_PASS'
-    secretName: null
+    password: notarypassword
+    # secretName: notary-secret <-- alternatively
+    # the secret data needs to look like this:
+    # data:
+    #   cred.yaml: b46enc(USER: notaryuser\nPASS: notarypassword)
+  # isAcr: true 
 ```
 
-For security in a real environment, you should first setup and then reference a [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) via `secretName`.
+If you simply want to verify non-official images on DockerHub, then just add the corresponding public key to the existing predefined notary configuration under `rootKeys`, next to the key for official images. But how do you get these public root keys?
 
-> At this point, Connaisseur only supports BasicAuth for connecting to the notary.
+#### Retrieving the public root key
 
-#### Set your public key as trust anchor
-
-> What is being done at this point is to set your personal root key as Connaisseur's trust anchor. If used in production, you may wish to go for [setting up delegation keys](https://docs-stage.docker.com/engine/security/trust/trust_delegation/#creating-delegation-keys) and keep the root key away from everyday-signing.
-
-Lastly, we need to configure the `notary.rootPubKey` that serves as a trust anchor and pins all signatures of deployed images to this public key. There is two different ways to get the required key:
-
-1. In case you just setup DCT and created your root keys, you should have a `root.pub` file in your folder. Remove the `role` and empty line and copy the contents to `notary.rootPubKey` in `helm/values.yaml`. The result should look similar to this (except for the different key value):
-
-```yaml
-  # the public part of the root key, for verifying notary's signatures
-  rootPubKey: |
-    -----BEGIN PUBLIC KEY-----
-    MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAETBDLAICCabJQXB01DOy315nDm0aD
-    BREZ4aWG+uphuFrZWw0uAVLW9B/AIcJkHa7xQ/NLtrDi3Ou5dENzDy+Lkg==
-    -----END PUBLIC KEY-----
-```
-
-2. In case you already had DCT setup or cannot find the file, we need to retrieve and convert the key manually from your previously created root key. For most shell users, there is a short script below. Otherwise, just follow the manual steps:
-
-##### manual
+For now let's assume you only want to verify image signatures you created/will create, using the key generated in step [two](#set-up-docker-content-trust). To get the public part of this you have to convert the key manually.
```

Comment:
```suggestion
For now let's assume you only want to verify image signatures you created/will create, using the key generated in [step two](#set-up-docker-content-trust). To get the public part of this you have to convert the key manually.
```

Reply:
resolved in 3b7d64cd6e6c5c09c2f7370d57ddb8a3b6c8869c